internal/backend/semtechudp/registry.go

```diff
@@ -18,7 +18,8 @@ var (
 
 // gatewayCleanupDuration contains the duration after which the gateway is
 // cleaned up from the registry after no activity
-var gatewayCleanupDuration = -1 * time.Minute
+// This value can be set in config [backend.SemtechUDP.cleanup_duration]
```

Comment:
I don't think these comments are needed.

Reply:
Forgot to remove them :-)